niederbayern_oberpfalz.py

```diff
@@ -111,88 +110,113 @@ def parse_url(url, today=False):
         'ZTVG':  'vegan',
         'ZTW':   'Wild'
     }
-    #canteen.setLegendData(legend)
 
+    # Create regular expressions for categories
     hg = re.compile("^HG[1-9]$")
     b = re.compile("^B[1-9]$")
     n = re.compile("^N[1-9]$")
 
-    #for w in 0, 1:
-    for w in [0]:
```

Comment:
I thought about it but then I removed it.
I will add it again because this way it should be better. Especially on monday morning.